packages/components/src/index.js

```diff
@@ -16,6 +16,7 @@ export { default as Animate } from './animate';
 export { default as AnglePickerControl } from './angle-picker-control';
 export { default as Autocomplete } from './autocomplete';
 export { default as BaseControl } from './base-control';
+export { default as __experimentalBoxControl } from './box-control';
```

Comment:
I wouldn't mind making this stable if we're confident with the API.

Reply:
Ohhh boy. My feeling is to see how it goes after we merge it. They may be some unexpected  updates we might need to make after receiving updates on padding (maybe).